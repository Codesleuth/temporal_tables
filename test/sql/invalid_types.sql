SET client_min_messages TO error;

CREATE TABLE invalid_types (a bigint, "b b" date, sys_period tstzrange);

CREATE TABLE invalid_types_history (a bigint, "b b" timestamp, sys_period tstzrange);

CREATE TRIGGER versioning_trigger
BEFORE INSERT OR UPDATE OR DELETE ON invalid_types
FOR EACH ROW EXECUTE PROCEDURE versioning('sys_period', 'invalid_types_history', true);

INSERT INTO invalid_types DEFAULT VALUES;

DELETE FROM invalid_types;