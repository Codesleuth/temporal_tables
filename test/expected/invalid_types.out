SET client_min_messages TO error;
CREATE TABLE invalid_types (a bigint, "b b" date, sys_period tstzrange);
CREATE TABLE invalid_types_history (a bigint, "b b" timestamp, sys_period tstzrange);
CREATE TRIGGER versioning_trigger
BEFORE INSERT OR UPDATE OR DELETE ON invalid_types
FOR EACH ROW EXECUTE PROCEDURE versioning('sys_period', 'invalid_types_history', true);
INSERT INTO invalid_types DEFAULT VALUES;
DELETE FROM invalid_types;
ERROR:  column "b b" of relation "invalid_types" is of type date but column "b b" of history relation "invalid_types_history" is of type timestamp without time zone
